### DISCUSSION TITLE
Possible to connect into an Electron compiled app instead of a Chromium based browser?

### DISCUSSION THREAD

**Discussion by Bolado** [go-rod/rod#440] [Q&A] [answered]
**Rod Version:** v0.99.0

## The code to demonstrate your question

```go
browser := rod.New()
launcher := launcher.New().Bin(execPath + "\\bin\\electronapp\\App.exe")
```

## What have you tried to solve the question
 I have seen it being done in some other libs, but I would like to know if it's possible on Rod, it's good to know because the motivation of it being supported is quite nice.

>  In comparison with the full-featured Chromium browser (~108MB Mac, ~113MB Linux, ~141MB Win for ZIP package), A portable alternative ------ Electron is able to handle most daily tasks but has half of Chromium's size (~55MB Mac, ~63MB Linux, ~58MB Win for ZIP package)

**Comment by ysmood** [comment, marked as answer]
According to here: https://www.electronjs.org/docs/api/command-line-switches#--remote-debugging-portport

```javascript
const { app } = require('electron')
app.commandLine.appendSwitch('remote-debugging-port', '8315')

app.whenReady().then(() => {
  // Your code here
})
```

Use rod to connect to the port:

```go
u := launcher.MustResolveURL("8315")
browser := rod.New().ControlURL(u).MustConnect()
```

For more info: https://github.com/go-rod/rod/blob/master/lib/examples/connect-browser/main.go
